locales/de.json

```diff
@@ -75,8 +75,8 @@
       }
     },
     "DiscardConfirmationModal": {
-      "title": "Verwerfen Sie alle nicht gespeicherten Änderungen",
-      "message": "Wenn Sie Änderungen verwerfen, werden alle Änderungen gelöscht, die Sie seit dem letzten Speichern vorgenommen haben.",
+      "title": "Verwirf alle nicht gespeicherten Änderungen",
```

Comment:
I checked these out from master, this won't show up in the diff once the base branch is rebased on master